sorachung/StoriesByUs
# Src setup - Project Setup

#### Changes Made

1. Added the whole project
   ​

#### Steps to Review

1. Checkout this branch locally.
   ```
   git fetch --all
   git checkout src-setup
   ```
2. Open a new Terminal tab (⌘T) and navigate to the server directory.
3. Test app functionality.
   > try logging in and logging out
4. View code file.
   > Confirm file modifications are present as indicated above.
   > Confirm no unused code or extraneous comments exist.
